packages/php-wasm/compile/php/Dockerfile

```diff
@@ -2143,6 +2143,9 @@ RUN set -euxo pipefail; \
 		# stream.stream_ops is sometimes undefined and the check fails. Let's adjust it to
 		# tolerate a null stream.stream_ops value.
 		/root/replace.sh "s/if\s*\(stream\.stream_ops\.poll\)/if (stream.stream_ops?.poll)/g" /root/output/php.js; \
+	# Emscriptend allows only directories to be mounted, but in Playground we support mounting files, directories, and symlinks.
+		# For file mounting to work, we need to remove the directory check.
+		/root/replace-across-lines.sh 's/(\s+)if\s*\(\s*!FS\.isDir\(node\.mode\)\s*\)\s*\{\s+throw\s+new\s+FS\.ErrnoError\(54\)\s*;\s+\}(\s+)/$1$2/gs' /root/output/php.js; \
```

Comment:
Are we certain this only affects that one instance? Maybe let's add a "--exactly-one-match" or similar flag to replace-across-lines.sh to make sure we're not replacing too much.